Formula/urbit.rb

```diff
@@ -0,0 +1,29 @@
+class Urbit < Formula
+  desc "Personal cloud computer"
+  homepage "https://urbit.org"
+  url "https://github.com/urbit/urbit/archive/v0.4.tar.gz"
+  sha256 "7320a3099b5a6392a6860f6824bb4b575bbde19ff1f59bc4661b3dbb9d1fbc69"
+
+  depends_on "gmp"
+  depends_on "libsigsegv"
+  depends_on "openssl"
+
+  depends_on "libtool" => :build
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+  depends_on "cmake" => :build
+
+  def install
+    ENV.deparallelize
+    system "make", "BIN=#{bin}", "LIB=#{share}"
+  end
+
+  test do
+    require "open3"
+    Open3.popen3("#{bin}/urbit") do |_, o, e, ps|
```

Comment:
Check and see if `assert_match "simple usage:", shell_output("#{bin}/urbit" 2>&1", 1)` works instead.